pgmpy/tests/test_models/test_BayesianModel.py

```diff
@@ -226,6 +226,11 @@ def test_remove_nodes_from(self):
         self.assertRaises(ValueError, self.G1.get_cpds, 'diff')
         self.assertRaises(ValueError, self.G1.get_cpds, 'grade')
 
+    def test__eq__(self):
+        model_copy = self.G1.copy()
+        self.assertTrue(self.G1.__eq__(model_copy))
```

Comment:
@Vaibhavdixit02 Tests are failing here. `self.G1.__eq__(model_copy)` is returning false.